ui-participant/src/hub/kit/KitTask.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { KitRequest } from '../../api/api'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faTruckFast } from '@fortawesome/free-solid-svg-icons'
+
+/** Renders kit tasks for the hub page */
+export default function KitTask({ kitRequests }: {kitRequests: KitRequest[]}) {
+  const kitEventProps = {
+    padding: '1em 0em',
+    borderBottom: '1px solid #e4e4e4',
+    width: '100%',
+    color: '#595959'
+  }
+
+  const kitHeaderProps = {
+    borderBottom: '1px solid #e4e4e4',
+    width: '100%',
+    color: '#595959'
+  }
+
```

Comment:
kitEventProps and kitHeaderProps are only used once, so I would inline them

Reply:
I figured this would look odd, and not surprisingly these are cut-and-paste from the kit card featrue I demonstrated. In that case, they make more sense, since they were reused a handful of times.
I kept them here for two reasons:

1. We may enhance this banner at some point: (e.g. Thank you for returning your kit....).
2. We may do something like the kit card feature at some point.

IOW, these are common stylings for kit info, and I'd like a way to pull them into one place. For the kit card development made playing with styling and keeping things consistent easier, and I think it helped readability of that code. These of course do not rise to a central style sheet level, but is there another construct that fulfills the use case I described?

Reply:
agree, that makes sense, so I'd pull them up out of the component along with the other stuff